### ISSUE TITLE
loss function wrong

### ISSUE THREAD

**Issue by LoveHRTF** [LoveHRTF/RCF-pytorch#3]

**Comment by YangJiao1996** [comment]
It's actually correct
